client/src/modules/CourseManagement/components/CourseEventModal/formState.ts

```diff
@@ -0,0 +1,87 @@
+import { message } from 'antd';
```

Comment:
Seems like it should be `useFormState` hook which could aggregate all logic regarding to this form